style-guide/template-style-guide.md

```diff
@@ -0,0 +1,131 @@
+<!-- Copy this Template. -->
```

Comment:
What we discovered in our first round of templates was that any comment (like this first line) are not visible to people who view the template from github.
As such, I recommend using another convention. As you are already using curly brackets, maybe use italic wrapped by square brackets. (Maybe also use italic for what is inside curly brackets too.)

Reply:
Sounds good. I've removed any comments and replaced them with {placeholder text}.